Doc/using/configure.rst

```diff
@@ -804,11 +804,14 @@ Libraries options
 
 .. option:: --with-system-libmpdec
 
-   Build the ``_decimal`` extension module using an installed ``mpdec``
-   library, see the :mod:`decimal` module (default is no).
+   Build the ``_decimal`` extension module using an installed ``mpdecimal``
+   library, see the :mod:`decimal` module (default is yes).
 
    .. versionadded:: 3.3
 
+   .. versionchanged:: 3.13
+      Default to using the installed ``mpdecimal`` library.
```

Comment:
Should we mention that a warning will be emitted if it's not found?

Reply:
I'm not sure. We normally don't emit warnings for missing dependencies; users are normally left to examining the `configure` output, and particularly the last lines where the extension module "summary" is printed.